dev-home-config.json

```
@@ -1,3 +1,3 @@
 {
-  "url": "exp://expo.io/@expo-home-dev/expo-home-dev-db2a78be19f92e56e0f7357dfe0d39a326c73ca3"
+  "url": "exp://expo.io/@expo-home-dev/expo-home-dev-481a6d30ded08a7f00630e7f6ecddb1325d778cb"
```

I know this was autogenerated but could you go ahead and make this change?
```suggestion
  "url": "exp://exp.host/@expo-home-dev/expo-home-dev-481a6d30ded08a7f00630e7f6ecddb1325d778cb"
```